### ISSUE TITLE
Convolutional Neural Networks for Sentence Classification

### ISSUE THREAD

**Issue by ghuijo** [One-Day-One-Paper/One-Day-One-Paper#173]
*''CNN trained on top of pre-trained word vectors for sentence-level classification tasks''*

### Model
* 모델 아키텍처
  <image src="https://user-images.githubusercontent.com/67627471/168474899-3c8ed2f1-a92f-4144-8b55-d24247aeb5dd.png" width="80%" height="80%" />

* 길이 n의 문장을 concatenation operator를 이용해 표현.
  <image src="https://user-images.githubusercontent.com/67627471/168475006-8d2a5377-ed5c-4fd2-94a4-70bc5518f9f4.png" width="30%" height="30%"/>

* 필터 w의 conv 연산으로 뽑은 피쳐 c
  <image src="https://user-images.githubusercontent.com/67627471/168475073-fcf8a627-7c26-456a-9c22-a3d5d8cf6158.png" width="30%" height="30%"/>

* max pooling 적용
  <image src="https://user-images.githubusercontent.com/67627471/168475125-5aae8cf9-d886-4860-86f2-931161bdd669.png" width="20%" height="20%" />

* one feature is extracted from one filter
  * the model uses multiple filters (with varying window sizes) to obtain multiple features
  * These features form the penultimate layer and are passed to a fully connected softmax layer
  whose output is the probability distribution over labels
* use the publicly available word2vec vectors that were trained on 100 billion words from Google News
  * Words not present in the set of pre-trained words are initialized randomly

### Datasets
* MR / SST-1 / SST-2 / Subj / TREC / CR / MPQA

### Results
* CNN과 다른 모델 결과 비교
  <image src="https://user-images.githubusercontent.com/67627471/168475369-375b5e8e-d5f7-45aa-bb74-161e35c3c0b2.png" width="70%" height="70%"/>
